### ISSUE TITLE
'chr' issue in converting bigWig files

### ISSUE THREAD

**Issue by joshtburdick** [liguowang/CrossMap#28]
I was attempting to convert a bigWig file from hg18 (but without "chr" in chromosome names) to GRCh37. Thus, I was using the 
`NCBI36_to_GRCh37.chain.gz liftOver` file. It looks like this should work with input files which include `chr`, or not. (And output chromosome names without `chr`, which I think is reasonable.)

I was getting an exception, the relevant bit of which is (I think) :
```
    OUT_FILE3.addEntries([r_chr], [int(r_st)], [int(r_end)], [float(r_value)]) 
RuntimeError: The entries you tried to add are out of order, precede already added entries, or otherwise use illegal values.
 Please correct this and try again.
```
Looking through the source, I saw that although this particular `liftOver` file was writing chromosome names without `chr`, the `.bgr `file was being written with `chr`, because of this bit of code in `lib/cmmodule.mapwig.py`:
```
...
  for k in sorted(taget_chrom_size.keys()):
    i_chrom = update_chromID('chr1',k)
```
I changed this omit the `chr`, to 
`    i_chrom = update_chromID('1',k)`
and my files seemed to convert without problems.

So I think this would fix that issue. Apologies that this isn't a pull request; I should really figure out how to do those...

Thank you for maintaining this tool.
Josh Burdick

**Comment by liguowang** [comment]
Thanks a lot, Josh! I will update the code and make a new release. 

Liguo

**Comment by liguowang** [comment]
Update: after a closer look. I think we should comment out the line 46 in mapwig.py:

#i_chrom = update_chromID('chr1',k)  

It will work for both "chr1" style and "1" style chromosome IDs.

**Comment by liguowang** [comment]
The bug is fixed in v0.5.3. Please let me know if you have any problems.

**Comment by joshtburdick** [comment]
Thanks for getting the code updated!
